nexuse2e-webapp/src/main/frontend/src/app/pipes/caps-to-title-case.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { TitleCasePipe } from "@angular/common";
+
+@Pipe({
+  name: "capsToTitleCase",
+  pure: true,
+})
+export class CapsToTitleCasePipe implements PipeTransform {
+  transform(value: unknown): unknown {
+    if (typeof value === "string") {
+      const titleCasePipe = new TitleCasePipe();
+      const uppercase = value.toUpperCase();
+      if (value === uppercase) {
+        return titleCasePipe.transform(value) as string;
+      } else {
+        return value as string;
```

Comment:
I think the `as` is not needed. typescript knows that value is here of the type string
```suggestion
      if (value === uppercase) {
        return titleCasePipe.transform(value);
      } else {
        return value;
```